data-service/src/main/java/uk/gov/gchq/palisade/service/data/request/AddSerialiserRequest.java

```diff
@@ -34,61 +35,71 @@
     private DataFlavour dataFlavour;
     private Serialiser<?> serialiser;
 
+    @Generated
     public AddSerialiserRequest dataFlavour(final DataFlavour dataFlavour) {
         requireNonNull(dataFlavour, "The data flavour cannot be set to null.");
```

Comment:
some of the requireNonNulls have a message, some dont, can this be standardised?

Reply:
All of the generated setters don't have messages - it's too messy to do string processing with velocity templates.
Some of the mutators do have messages as they were already there.
When you pass a null value to these setters, it's pretty obvious what went wrong - there's one test on the line for one argument etc. etc.
Definitely worth using the messages in constructors where you might pass 5 or 6 arguments, but in these cases I don't think the message really adds much value

Reply:
My gut says to have the message, but I see what you mean if it errors there then you will get the stacktrace that includes the class and line number of the issue. In which case the message is redundant. Therefore I don't mind as long as we are consistent

Reply:
The only change would be to mutators, which we haven't standardised under this PR